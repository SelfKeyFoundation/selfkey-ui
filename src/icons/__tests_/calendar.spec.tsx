import * as React from 'react';
import { create } from 'react-test-renderer';
import { CalendarIcon } from '../calendar';

describe('Calendar  Icon', () => {
  it('should show Calendar Icon', () => {
    const component = create(
      <CalendarIcon/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});





