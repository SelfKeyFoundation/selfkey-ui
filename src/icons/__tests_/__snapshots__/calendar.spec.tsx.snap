// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar  Icon should show Calendar Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root-1 Component-root-0-1-1"
  focusable="false"
  role="presentation"
  version="1.1"
  viewBox="0 0 20 20"
>
  <g
    fill="none"
    fill-rule="evenodd"
    id="ðŸ’»-ID-Dashboard"
    stroke="none"
    stroke-width="1"
  >
    <g
      fill="#23E6FE"
      id="405.1---Edit-Attribute-+-Date-of-Birth"
      transform="translate(-1035.000000, -347.000000)"
    >
      <g
        id="Modal"
        transform="translate(330.000000, 200.000000)"
      >
        <g
          id="address-1"
          transform="translate(40.000000, 109.000000)"
        >
          <path
            d="M682.971246,46 L666.996805,46 L666.996805,42.971 C666.996805,42.435 667.43111,42 667.966254,42 L668.99361,42 L668.99361,44 L670.990415,44 L670.990415,42 L678.977636,42 L678.977636,44 L680.974441,44 L680.974441,42 L681.972843,42 C682.523962,42 682.971246,42.448 682.971246,43 L682.971246,46 Z M682.971246,55 C682.971246,55.55 682.521965,56 681.972843,56 L667.995208,56 C667.444089,56 666.996805,55.552 666.996805,55 L666.996805,48 L682.971246,48 L682.971246,55 Z M665.063898,55.761 C665.063898,56.865 666.021366,58 667.123602,58 L683.098043,58 C684.201278,58 685,56.979 685,55.761 C685,55.372 684.968051,43.36 684.968051,42.708 C684.968051,40.626 684.6875,40 680.974441,40 L680.974441,38 L678.977636,38 L678.977636,40 L670.990415,40 L670.990415,38 L668.99361,38 L668.99361,40 L666.996805,40 C665.898562,40 665,40.9 665,42 L665.063898,55.761 Z"
            id="icon-calendar"
          />
        </g>
      </g>
    </g>
  </g>
</svg>
`;
